sdk/identity/identity/test/msalTestUtils.ts

```diff
@@ -133,6 +133,11 @@ export async function msalNodeTestSetup(
   await recorder.addSanitizers(
     {
       bodySanitizers: [
+        {
+          regex: true,
+          target: `client_secret=[a-zA-Z0-9-._]*`,
+          value: "client_secret=azure_client_secret",
```

Comment:
Error being..
```
[node-tests] Request and record bodies do not match at index 378:
[node-tests]      request: "nt_secret=azure_client_secret&"
[node-tests]      record:  "nt_secret=s%3F%3Dq2ZAdO%3D-Vn8"
```
regex should also include % in there perhaps.

Reply:
done